src/pages/payment-methods/index.tsx

```diff
@@ -416,4 +416,4 @@ const PaymentMethods = ({ locale }: PaymentMethodsProps) => {
     )
 }
 
-export default WithIntl()(PaymentMethods)
+export default PaymentMethods
```

Comment:
don't we need this ? :thinking:

Reply:
We need this, fixed in the new commit. Thanks